### ISSUE TITLE
False positive with `no-array-prototype-extensions` when using `reject()`

### ISSUE THREAD

**Issue by fengb** [ember-cli/eslint-plugin-ember#1548]
The method `RSVP.defer.reject()` is triggering false positives:
```js
import { defer } from 'rsvp';

const deferred = defer();
deferred.reject();
```

<img width="644" alt="image" src="https://user-images.githubusercontent.com/219422/182179205-760748c7-0c12-41ec-800b-2378159c9e61.png">

**Comment by bmish** [comment]
Looks like this is a common pattern we could ignore:

https://api.emberjs.com/ember/release/functions/rsvp/defer
https://stackoverflow.com/a/24314453
